Library/Homebrew/cmd/man.rb

```diff
@@ -25,21 +25,31 @@ def man
       puts "Writing HTML fragments to #{DOC_PATH}"
       puts "Writing manpages to #{TARGET_PATH}"
 
-      Dir["#{SOURCE_PATH}/*.md"].each do |source_file|
-        args = %W[
-          --pipe
-          --organization=Homebrew
-          --manual=brew
-          #{source_file}
-        ]
-        page = File.basename(source_file, ".md")
-
-        target_html = DOC_PATH/"#{page}.html"
-        target_html.atomic_write Utils.popen_read("ronn", "--fragment", *args)
-
-        target_man = TARGET_PATH/page
-        target_man.atomic_write Utils.popen_read("ronn", "--roff", *args)
-      end
+      header = IO.read("#{SOURCE_PATH}/header.1.md")
+      footer = IO.read("#{SOURCE_PATH}/footer.1.md")
```

Comment:
That's a bit of a bike-shedding argument, but I'd prefer the files to be named `brew.1.header.md` and `brew.1.footer.md`. Also let's make better use of `Pathname` (our preferred way when working with paths):

```ruby
header = (SOURCE_PATH/"brew.1.header.md").read
footer = (SOURCE_PATH/"brew.1.footer.md").read
```